contributing.md

```diff
@@ -7,22 +7,44 @@ The Drupal Media Team has been growing over the last few years and gradually gai
+
+We also encourage teams that are hostig Drupal events to organize their own media sprints. Doing so is fairly straightforward as you can select issues from the cue and then organize an effort to work through them (feel free to reach out to the media team with questions about priorities to work on). For an example, see the [2015 Zurich Media sprint] (https://groups.drupal.org/node/488488) (see [blog post](http://www.md-systems.ch/en/blog/2015-10/join-the-drupal-8-media-initiative)).
+
 ##Documentation
 
-You can also contribute directly on our [guide repository](https://github.com/drupal-media/d8-guide/issues) by editing the files through the [GitHub](https://github.com/) interface directly in your browser. Alternatively, you can clone the repository and edit the guide in your favorite text editor or in the official [GitBook desktop app](https://www.gitbook.com/editor).
+We could almost always use help with writing new documentation and reviewing/updating existing documentation. Development skill are often not needed for this, as contributing in this way often requires just downloading/installing a module and then using it to see how it's configured and works. After which, you can likely be of assistance in either writing or updating/reviewing documentation.
 
-All contributions are welcome and greatly appreciated.
+You can easily contribute directly on our Drupa Media [guide repository](https://github.com/drupal-media/d8-guide/issues) by editing the files through the [GitHub](https://github.com/) interface directly in your browser. Alternatively, you can clone the repository and edit the guide in your favorite text editor or in the official [GitBook desktop app](https://www.gitbook.com/editor).
 
+All contributions are welcome and greatly appreciated.
 
 ##Organizing & Project Management
 
-*FPO/TBA - This content needs to be written, see our [Contributing](contributing.md) section get involved.*
-
+The vast majority of the current Drupal Media team are developers, and primarily back-end modules developers. We currently don't have a program manager on the team, and most of our organizing efforts are being handled by developers in addition to or in lieu of their development efforts. Simply put, we could really using assistance on the this front from new team members, with organizational, communications or project management skills. Apart from freeing up developer time to develop, it would help our team collaborate better and stay well aligned if we had more of this sort of assistance (e.g., helping summarize the roadmap, preparing quarterly progress updates, and manage our often interdependent development tasks/projects). Please reach out to the team if you're interested in contributing this sort of invaluable assistance).
```

Comment:
s/we could really using/we could really use?
Last closing bracket is not needed.